roles/matrix-cactus-comments/templates/cactus_appservice.yaml.j2

```diff
@@ -0,0 +1,19 @@
+# A unique, user-defined ID of the application service which will never change.
+id: "Cactus Comments"
+
+# Where the cactus-appservice is hosted:
+url: "http://matrix-cactus-comments:{{ matrix_cactus_comments_container_port }}"
+
+# Unique tokens used to authenticate requests between our service and the
+# homeserver (and the other way). Use the sha256 hashes of something random.
+# CHANGE THESE VALUES.
+as_token: "{{ matrix_cactus_comments_as_token }}"
+hs_token: "{{ matrix_cactus_comments_hs_token }}"
+
+# User associated with our service. In this case "@cactusbot:example.com"
+sender_localpart: "{{ matrix_cactus_comments_user_id }}"
```

Comment:
The comment here is confusing. It makes it look like we full MXID is expected, even though it actually says `_localpart` and we're using a user id (localpart) as well.

Maybe it's how it is in the upstream configuration though? If so, we may leave it.

Reply:
Comment is corrected